CMakeLists.txt

```diff
@@ -144,10 +144,21 @@ if (MSVC)
     include_directories (${BOOST_ROOT})
     link_libraries( bcrypt.lib )
 else()
-    set(BOOST_INCLUDEDIR ${BOOST_ROOT})
-    set(BOOST_LIBRARYDIR ${BOOST_ROOT}/stage/lib)
-    find_package(Boost COMPONENTS coroutine filesystem system REQUIRED)
-    link_libraries(Boost::coroutine Boost::filesystem Boost::system)
+    option(BEAST_ALTERNATE_PATH "" ON)
```

Comment:
What is this funny business?

Reply:
Helps my workflow without damaging yours